fw/str_avx2.S

```diff
@@ -195,13 +195,13 @@ dbg_prefix_vec:
 	push	%r9
 	push	%r10
 	push	%r11
-	push	%elfags
+	pushfq
```

Comment:
According to the Intel manual vol 2, this is pushing `RFLAGS` with instruction `PUSHFQ`, why normal `PUSHF` isn't used?

Reply:
I use pushfq since we are in 64 bit mode, but since you right  PUSHF is enough. Fix.